src/GeositeFramework/plugins/launchpad/style.css

```diff
@@ -26,3 +26,41 @@
     list-style-type: none;
     padding: 0;
 }
+
+.launchpad-plugin .plugins .button.truncate-toggle, .scenarios .button.truncate-toggle {
+    padding: 2px !important;
+    font-size: 0.7em !important;
+    width: 35%;
+    float: none;
+}
+
+.truncate-box {
+    overflow: hidden;
+    position: relative;
+    line-height: 1.2em;
+    max-height: 2.6em;
+    text-align: justify;
+    margin-right:-1em;
+    padding-right: 2em;
+}
+
+.truncate-container > p {
+    width: 90%;
+}
+
+.truncate-box:before {
+    content: '(...)';
+    position: absolute;
+    right: 5px;
+    bottom: 2px;
+}
+
+.truncate-box:after {
```

Comment:
I removed this `.truncate-box:after` block during testing and it made no visual difference. Consider removing.